src/imperative/cached_op.h

```diff
@@ -547,6 +558,7 @@ class CachedOp {
       const std::vector<NDArray*>& inputs,
       const std::vector<OpReqType>& reqs,
       const std::vector<NDArray*>& outputs);
```

Comment:
Can we add some comment for this method?

Reply:
Which method are you referring to?  Sorry, it's not obvious from the quoted code fragment (probably because I messed commit history a bit? - now fixed).